city_scrapers/spiders/chi_ssa_21.py

```diff
@@ -0,0 +1,161 @@
+
+            yield data
+
+    def _parse_name(self, item):
+        """
+        Parse or generate event name.
+        """
+        return 'Commission Meetings'
+
+    def _parse_description(self, item):
+        """
+        Parse or generate event description.
+        """
+        description = 'N/A'
+
+        # The itinerary of the meeting is always stored in the <ul>
+        # element immediately following
+        detailElement = item.xpath('following-sibling::*[1]')
+        name = detailElement.xpath('name()').extract_first()
+
+        if (name == 'ul'):
+            topics = list(map(
+                lambda topic: ''.join(filter(
+                    # Remove any strings that are empty
+                    None,
+                    [
+                        # Title of topic
+                        ''.join(topic.xpath('strong/text()').extract()).strip(),
+                        # Detail of topic
+                        '[' + ''.join(topic.xpath('text()').extract()).strip() + ']'
+                    ]
+                )),
+                detailElement.xpath('li')
+            ))
+
+            description = ':'.join(topics)
+
+        return description
+
+    def _parse_classification(self, item):
+        """
+        Parse or generate classification (e.g. public health, education, etc).
+        """
+        return COMMISSION
+
+    def _parse_start(self, item):
+        """
+        Parse start date and time.
+        """
+        startTime = self._parse_date(item)
+        startTime = startTime.replace(hour=9, minute=0)
+
+        ret = {
+            'date': startTime.date(),
+            'time': startTime.time(),
+            'note': None
+        }
+
+        return ret
+
+    def _parse_end(self, item):
+        """
+        Parse end date and time.
+        """
+        endTime = self._parse_date(item)
+        endTime = endTime.replace(hour=11, minute=0)
+
+        ret = {
+            'date': endTime.date(),
+            'time': endTime.time(),
+            'note': 'estimated 2 hours after start time'
+        }
+
+        return ret
+
+    def _parse_date(self, item):
+        rawDate = item.xpath('strong/text()').extract_first()
+        return dateutil.parser.parse(rawDate)
+
+    def _parse_all_day(self, item):
+        """
+        Parse or generate all-day status. Defaults to False.
+        """
+        return False
+
+    def _parse_location(self, item):
+        """
+        Parse or generate location. Latitude and longitude can be
+        left blank and will be geocoded later.
+        """
+
+        defaultLocation = 'Bistro Campagne, 4518 N. Lincoln Avenue'
+
+        # If location has changed, this is where it is noted
+        location = ''.join(
+            item.xpath('em//text()').extract()
+        ).strip()
+
+        if not location:
+            location = defaultLocation
+
+        return {
+            'address': location,
+            'name': '',
```

Comment:
When possible, we should split the name "Bistro Campagne" from the address and include it here. It looks like they're relatively consistently split by a comma, so we could check for that above.

Also, since we can assume this will always be in Chicago, we should add "Chicago, IL" to the end of the address if it isn't already present. That helps the accuracy of our geocoding later on